main.lsf

```diff
@@ -160,7 +160,11 @@ Static main
 	Object asm/unk_0203BC10.o
 	Object asm/unk_0203DB6C.o
 	Object src/field_system.o
-	Object asm/unk_0203E348.o
+	Object src/field/location_backup.o
```

Comment:
not in the field overlay so should not be in the field folder

Reply:
Imo the file hierarchy should group things together by functionality, not necessarily by rom position. The `battle/` directory is already doing this (e.g. `src/battle/battle_setup.c` is in the main overlay).

Reply:
battle_setup is in the wrong place @adrienntindall >:(
please don't put stuff like this in the field folder, there is a reason for this (i.e. knowing which overlay must be loaded to access funcs in this folder)

Reply:
I do what I want

Reply:
Moved files into `src/`